lib/msal-core/src/UserAgentApplication.ts

```diff
@@ -1422,7 +1305,9 @@ export class UserAgentApplication {
     private renewToken(scopes: Array<string>, resolve: Function, reject: Function, account: Account, serverAuthenticationRequest: ServerRequestParameters): void {
         const scope = scopes.join(" ").toLowerCase();
         this.logger.verbose("renewToken is called for scope:" + scope);
-        const frameHandle = this.addHiddenIFrame("msalRenewFrame" + scope);
+
+        const frameName = `msalRenewFrame${scope}`;
```

Comment:
Why are we adding scope? Is it to distinguish between renew calls as we keep the iframe open? I have always had my doubts on using scope as a distinguisher as the developer can make sequential calls to acquireTokenSilent with the same scopes. Can you please check this?

Reply:
I think it just reuses the same iframe for calls with the same scopes. Not clear why.